src/components/Dashboard/Menu/Item.tsx

```diff
@@ -23,12 +23,15 @@ export const DashboardMenuItem = ({
     as={route ? ReactRouterLink : undefined}
     to={route ? generatePath(route) : undefined}
     onClick={hasChildren ? onToggle : undefined}
-    isActive={isActive} // Set active state
+    isActive={hasChildren ? (isOpen ? true : false) : false} // Set active state
```

Comment:
Why...? The logic for being or not active was done in Options, why are you redoing the logic here again? If it wasn't working as expected, should have been fixed where it was being done.